src/frontend/src/App.vue

```diff
@@ -1,29 +1,19 @@
 <template>
   <div id="app">
     <div class="main">
-      <div class="main__wrapper">
-        <div class="main__header">
-          <img
-            src="@/assets/img/logo.svg"
-            width="300"
-            height="47"
-            alt="V!U!E! Pizza"
-          />
-        </div>
-        <h1>Добро пожаловать!</h1>
-        <p>
-          Это проект V!U!E! Pizza для обучения на профессиональном
-          онлайн‑курсе<br />
-          <b>«Vue.js для опытных разработчиков».</b>
-        </p>
-      </div>
+      <mainPage></mainPage>
```

Comment:
Исходя из комментария к 10-16 строкам
```suggestion
      <Index></Index>
```